src/mon/PGMap.cc

```diff
@@ -955,16 +955,17 @@ void PGMapDigest::dump_object_stat_sum(
         tbl << "N/A";
       else
         tbl << stringify(si_u_t(pool->quota_max_objects));
-
       if (pool->quota_max_bytes == 0)
         tbl << "N/A";
       else
         tbl << stringify(byte_u_t(pool->quota_max_bytes));
-
-      tbl << stringify(si_u_t(sum.num_objects_dirty))
-	  << stringify(byte_u_t(statfs.data_compressed_allocated))
-	  << stringify(byte_u_t(statfs.data_compressed_original))
-	  ;
+      if (pool->is_tier()) {
+        tbl << stringify(si_u_t(sum.num_objects_dirty))
+	      << stringify(byte_u_t(statfs.data_compressed_allocated))
+	      << stringify(byte_u_t(statfs.data_compressed_original));
```

Comment:
why would we include these in the if block?

Reply:
sorry, missed those fields, thanks for review